test/jest/Orderbook.spec.ts

```diff
@@ -90,8 +92,15 @@ describe('OrderBook', () => {
       port: 9735,
     }, network);
     db = new DB(loggers.db, config.dbpath);
-    pool = new Pool(config.p2p, config.network, loggers.p2p, db.models, '1.0.0');
-    swapClientManager = new SwapClientManager(config, loggers, pool);
+    pool = new Pool({
+      config: config.p2p,
+      xuNetwork: config.network,
+      logger: loggers.p2p,
+      models: db.models,
+      version: '1.0.0',
+      nodeKey: new NodeKey(new Buffer(1)),
```

Comment:
We should be able to avoid passing arguments to `NodeKey` since it's mocked.

Consider casting NodeKey to any and then to `jest.Mock`. Example:
https://github.com/ExchangeUnion/xud/blob/master/test/jest/HttpService.spec.ts#L5

Reply:
Sounds good, I'll try this out.